modules/bookCollection.js

```diff
@@ -0,0 +1,54 @@
+class BookCollection {
+  constructor(bookArray) {
+    this.bookArray = bookArray;
+  }
+
+  displayBooks() {
+    const bookContainer = document.querySelector('.books-container');
+    bookContainer.innerHTML = '';
+    this.bookArray.forEach((book) => {
+      bookContainer.insertAdjacentHTML(
+        'beforeend',
+        `
+        <div class="book-details" id="book-${book.id}">
+          <p>"${book.title}" by ${book.author}</p> 
```

Comment:
- [ ] Hi please as mentioned in the requirements, you are supposed to refactor your code to arrow functions so, please change your components from class to function